### PR TITLE
fix: Support very large findings

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments

Based on the way we were constructing the logs previously, if the "message" (i.e. the JSON-formatted logs from a security tool) contained more than 2,092,841 "arguments" (this varies depending on the platform and OS; see additional details [here](https://www.in-ulm.de/~mascheck/various/argmax/#results)) then bash would fail because this exceeds `ARG_MAX`.  We moved to a file-based constructor approach which should avoid this issue.

 - [X] Add a test.  Added [here](https://github.com/SeisoLLC/easy_infra/pull/235/files#diff-002b17638e4a840ab803fb150dddd393a637a14314ed5c395a048192850b2603R396-R408) and confirmed it worked via [this TDD test](https://github.com/SeisoLLC/easy_infra/actions/runs/4026535963/jobs/6921175762).  It did work, but did not fail because we purposefully don't fail on being unable to generate logs.
 - [X] Update the test to actually fail if it detects ERROR in the output.  Done [here](https://github.com/SeisoLLC/easy_infra/actions/runs/4032142605/jobs/6931842854#step:11:18).

## Pull Request Checklist

Thank you for submitting a contribution to `easy_infra`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:

- [X] Rebase your branch against the latest commit of the target branch
- [X] If you are adding a dependency, please explain how it was chosen
- [X] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results
- [X] If there is an issue associated with your Pull Request, link the issue to the PR.

### CODE REVIEW

**Review by ChiefHolland — APPROVED** [review]
LGTM.

**Comment by JonZeolla** [comment]
Hmm, not sure why my TDD failed, need to look into that prior to final approval/merge

**Comment by JonZeolla** [comment]
Working now. Just needed some minor tweaks to the test

File: build/common.sh [hunk 1]
```diff
@@ -58,69 +100,71 @@ function _log() {
...
+    _log "easy_infra.stdouterr" info unknown "easy_infra" "${label_cwd}" string "Failed to generate a valid JSON log message"
   fi
-  echo "${LOG_MESSAGE}" &>> /var/log/easy_infra.log
 }
 
 
 function _feedback() {
-  timestamp="$(date -Ins)"
+  local timestamp
+  timestamp="$(date --iso-8601=seconds --utc)"
```
Comment (JonZeolla) [inline build/common.sh:155]: This makes the timestamps consistent between feedback and logs, and also shortens the feedback (what goes to stdout/err)

File: tasks.py [hunk 2]
```diff
@@ -605,8 +605,6 @@ def build(_c, tool="all", environment="all", trace=False, debug=False, dry_run=F
                 f"Would have run utils.render_jinja2 on {constants.FUNCTIONS_INPUT_FILE}..."
             )
 
-        # TODO: Figure out how to handle -large, somewhere in this function?
```
Comment (JonZeolla) [inline tasks.py:608]: We decided not to do a -large image right now.  Keep the images small and focused

**Review by digeseiso — APPROVED** [review]
LGTM

**Pull request merged**